src/frontend/src/lib/canisters/backend.canister.ts

```diff
@@ -161,4 +162,10 @@ export class BackendCanister extends Canister<BackendService> {
 
 		return response.Ok;
 	};
+
+	allowSigning = async (): Promise<AllowSigningResponse> => {
+		const { allow_signing } = this.caller({ certified: true });
+
+		return allow_signing();
```

Comment:
Shall we handle possible response errors here, similar to other functions? Or in this particular case you need the whole `AllowSigningResponse`?

Reply:
Yes but I'll do it in another PR given that I provided this PR before you introduced the pattern.

Reply:
Got it